site/en/blog/fugu-showcase/index.md

```diff
@@ -34,6 +36,40 @@ Easter egg, the Project Fugu API Showcase is of course
 <a href="https://tomayac.github.io/fugu-showcase/data/#tomayac.github.io!fugu-showcase!data" target="showcase">contained
 in the Project Fugu API Showcase</a>. Happy browsing!
 
-<div style="height: 100%; width: 100%;">
-  <iframe title="Fugu showcase" name="showcase" style="min-height: 800px; width: 100%; border: 0;" src="https://tomayac.github.io/fugu-showcase/data/" allow="web-share; clipboard-write; clipboard"></iframe>
+## Showcase
+
+<div class="showcase-container" style="height: 100%; width: 100%">
+  <iframe
+    title="Fugu showcase"
+    name="showcase"
+    style="min-height: 1000px;width: 100%;border: solid 1px var(--color-hairline);"
+    src="https://tomayac.github.io/fugu-showcase/data/"
+    allow="web-share; clipboard-write; clipboard"
+    onload="this.contentWindow.postMessage({hash:location.hash.substr(1),search:location.search.substr(1)},'*');"
+  ></iframe>
 </div>
+
+<script>
```

Comment:
I'm rather sure this will violate CSP policy

Reply:
Perfect then, resolving.